spinnaker_camera_driver/src/SpinnakerCamera.cpp

```diff
@@ -337,14 +346,16 @@ void SpinnakerCamera::grabImage(sensor_msgs::Image* image, const std::string& fr
 
       if (image_ptr->IsIncomplete())
       {
-        throw std::runtime_error("[SpinnakerCamera::grabImage] Image received from camera " + std::to_string(serial_) +
+        ROS_WARN_STREAM("[SpinnakerCamera::grabImage] Image received from camera " + std::to_string(serial_) +
                                  " is incomplete.");
+        return false;
       }
       else
       {
         // Set Image Time Stamp
-        image->header.stamp.sec = image_ptr->GetTimeStamp() * 1e-9;
-        image->header.stamp.nsec = image_ptr->GetTimeStamp();
+        ros::Time imgStamp(image_ptr->GetTimeStamp() * 1e-9);
+        ros::Duration ptpOffset(-37);
```

Comment:
comment on the -37 (tai/utc offset is not accounted for in sensor output, apparently).